### PR TITLE
Feature/profile flag

### PR DESCRIPTION
**Pull request by NullableInt**
Adds profile flag so I can run profiling to see if we can speed up the process.

### CODE REVIEW

File: .docker/entrypoint/lib/context.rb [hunk 1]
```diff
@@ -15,11 +15,12 @@ class Context
       attr_reader :var_dir
       attr_reader :data_dir
       attr_reader :debug
+      attr_reader :profile
       attr_reader :auth_token
       attr_reader :configuration
       attr_reader :arguments
 
-      def initialize(env, var_dir, data_dir, auth_token = nil, debug = false)
+      def initialize(env, var_dir, data_dir, auth_token = nil, debug = false, profile = false)
```
Comment (payex-dev) [inline .docker/entrypoint/lib/context.rb:23]: Codacy found an issue: [Line is too long. [94/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

Comment (payex-dev) [inline .docker/entrypoint/lib/context.rb:23]: Codacy found an issue: [Avoid parameter lists longer than 5 parameters. [6/5]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

File: .docker/entrypoint/lib/context.rb [hunk 2]
```diff
@@ -41,10 +43,14 @@ def arguments=(args)
         args.must_be_a! Arguments
         @arguments = args
       end
-
+
```
Comment (payex-dev) [inline .docker/entrypoint/lib/context.rb:46]: Codacy found an issue: [Trailing whitespace detected.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

File: .docker/entrypoint/lib/entrypoint.rb [hunk 3]
```diff
@@ -30,8 +30,9 @@ def initialize_context(context)
         jekyll_data_dir = ENV.fetch('JEKYLL_DATA_DIR', Dir.pwd)
         jekyll_var_dir = ENV.fetch('JEKYLL_VAR_DIR')
         debug = ENV.fetch('DEBUG', false)
+        profile = ENV.fetch('PROFILE', false)
 
-        Context.new(env, jekyll_var_dir, jekyll_data_dir, auth_token, debug)
+        Context.new(env, jekyll_var_dir, jekyll_data_dir, auth_token, debug, profile)
```
Comment (payex-dev) [inline .docker/entrypoint/lib/entrypoint.rb:35]: Codacy found an issue: [Line is too long. [85/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

File: .docker/entrypoint/lib/context.rb [hunk 4]
```diff
@@ -41,10 +43,18 @@ def arguments=(args)
...
         @arguments = args
       end
-
+      
       def verbose?
         @debug || arguments.log_level == :debug
       end
+      
+      def profile?
+        puts "Profile?"
```
Comment (payex-dev) [inline .docker/entrypoint/lib/context.rb:52]: Codacy found an issue: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

File: .docker/entrypoint/lib/context.rb (hunk 1 above)
Comment (payex-dev) [inline .docker/entrypoint/lib/context.rb:23]: Codacy found an issue: [Line is too long. [94/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

File: .docker/entrypoint/lib/context.rb [hunk 5]
```diff
@@ -41,10 +43,18 @@ def arguments=(args)
         args.must_be_a! Arguments
         @arguments = args
       end
-
+
```
Comment (payex-dev) [inline .docker/entrypoint/lib/context.rb:46]: Codacy found an issue: [Trailing whitespace detected.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

File: .docker/entrypoint/lib/context.rb (hunk 1 above)
Comment (payex-dev) [inline .docker/entrypoint/lib/context.rb:23]: Codacy found an issue: [Avoid parameter lists longer than 5 parameters. [6/5]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

File: .docker/entrypoint/lib/commands/jekyll_server.rb [hunk 6]
```diff
@@ -15,6 +15,9 @@ module Commands
       class JekyllServer < JekyllBuilder
         def execute
           super
+          puts "<<<------------->>>"
```
Comment (payex-dev) [inline .docker/entrypoint/lib/commands/jekyll_server.rb:18]: Codacy found an issue: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

File: .docker/entrypoint/lib/entrypoint.rb (hunk 3 above)
Comment (payex-dev) [inline .docker/entrypoint/lib/entrypoint.rb:35]: Codacy found an issue: [Line is too long. [85/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

File: .docker/entrypoint/lib/commands/jekyll_server.rb (hunk 6 above)
Comment (payex-dev) [inline .docker/entrypoint/lib/commands/jekyll_server.rb:18]: Codacy found an issue: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

File: .docker/entrypoint/lib/context.rb [hunk 7]
```diff
@@ -41,10 +41,17 @@ def arguments=(args)
...
```
Comment (payex-dev) [inline .docker/entrypoint/lib/context.rb:44]: Codacy found an issue: [Trailing whitespace detected.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

File: .docker/entrypoint/lib/context.rb [hunk 8]
```diff
@@ -41,10 +41,17 @@ def arguments=(args)
...
         @arguments = args
       end
-
+      
       def verbose?
         @debug || arguments.log_level == :debug
       end
+      
+      def profile?
+        puts "Profile?"
```
Comment (payex-dev) [inline .docker/entrypoint/lib/context.rb:50]: Codacy found an issue: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

File: .docker/entrypoint/lib/context.rb [hunk 9]
```diff
@@ -45,6 +45,10 @@ def arguments=(args)
       def verbose?
         @debug || arguments.log_level == :debug
       end
+
```
Comment (payex-dev) [inline .docker/entrypoint/lib/context.rb:48]: Codacy found an issue: [Trailing whitespace detected.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

Comment (payex-dev) [inline .docker/entrypoint/lib/context.rb:48]: Codacy found an issue: [Trailing whitespace detected.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)

**Review by asbjornu — APPROVED** [review]
👏

**Pull request merged**